### DISCUSSION TITLE
How to inherit the modal.blade.php file

### DISCUSSION THREAD

**Discussion by mikecchops** [wire-elements/modal#409] [Q&A] [answered]
Hi, we are looking to make some changes to the file:  `/wire-elements/modal/resources/views/modal.blade.php`

I've tried copying the modal.blade.php file to `/resources/views/vendor/wire-elements/modal.blade.php` - but it is still using the first file. 

Does anyone know how we can specify a new modal.blade.php file? Thanks

**Comment by JulianGlueck** [comment]
Did you find out anything in the meantime? I am facing the same issue since updating from Livewire 2 to Livewire 3.

**Comment by PhiloNL** [comment, marked as answer]
It should work, have you tried the `php artisan vendor:publish` command?

**Comment by JulianGlueck** [comment]
Thanks a lot Philo; I missed, that the folder name changed. I got it to work by moving `/resources/views/vendor/livewire-ui-modal/modal.blade.php` to `/resources/views/vendor/wire-elements-modal/modal.blade.php`. 🥳
